src/components/dashboard/components/dictionary/DictionaryContainer.jsx

```diff
@@ -23,11 +23,15 @@ export class DictionaryOverview extends Component {
     fetchDictionaryConcepts: propTypes.func.isRequired,
     fetchVersions: propTypes.func.isRequired,
     createVersion: propTypes.func.isRequired,
-    error: propTypes.array.isRequired,
+    error: propTypes.array,
     releaseHead: propTypes.func.isRequired,
     isReleased: propTypes.bool.isRequired,
   };
 
+  static defaultProps = {
+    error: null,
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, this is part of the first bullet point that handles restructuring of the code in order for it to be tested. After testing, I got warnings on the terminal console and solved it by giving the prop a default value.